fmc/src/flow/rt_alias.rs

```diff
@@ -44,26 +44,25 @@ impl RtAliasLayer {
         hand_off: &mut HandOff,
         input: &DiceInput,
     ) -> CaliptraResult<DiceOutput> {
+        if Self::kv_slot_collides(input.cdi) {
```

Comment:
Can collision detection be a compile time assertion?

Reply:
Yes, but I think that only works if FMC ignores the key handle number it observes in the handoff table and instead uses a hard-coded value that was compile-time-asserted to be unique from the values used by ROM.

Reply:
We could make it harder to accidentally introduce collisions though, if we move the key IDs defined [in ROM](https://github.com/chipsalliance/caliptra-sw/blob/2be0e16e6832400053e67069f9f89989a530958c/rom/dev/src/flow/cold_reset/mod.rs#L34-L39) and [in FMC](https://github.com/chipsalliance/caliptra-sw/blob/2be0e16e6832400053e67069f9f89989a530958c/fmc/src/flow/mod.rs#L29-L30) into one file in the common directory. Could take care of this in a follow-up.

Reply:
I like the idea.